Classes/ExplorerInterface/FLEXExplorerViewController.m

```diff
@@ -835,6 +835,19 @@ - (BOOL)wantsWindowToBecomeKey
     return self.previousKeyWindow != nil;
 }
 
+- (void)toggleToolWithViewControllerProvider:(UIViewController *(^)())future completion:(void(^)())completion
+{
+    void (^presentBlock)() = ^{
+        [self makeKeyAndPresentViewController:future() animated:YES completion:completion];
```

Comment:
Slightly nicer to skip the block declaration and move this line into the `else` path

Reply:
Didn't even notice that the block wasn't being used in multiple places anymore. Definitely.